Assets/MRTK.ThirdParty/MRTK-Quest/Scripts/Utils/HandPoseUtils.cs

```diff
@@ -90,5 +94,98 @@ public static bool IsThumbGrabbing(Handedness hand)
             }
             return false;
         }
+
+        /*
+        * Finger Curl Utils: Util Functions to calculate the curl of a specific finger. 
+        * Author: Chaitanya Shah
+        * github: https://github.com/chetu3319
+        */
+
```

Comment:
Please remove one white space here. Try not to keep more than one blank line before spaced out areas here.